modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/data/handler/DDMFormAdminPortletDataHandler.java

```diff
@@ -225,13 +226,21 @@ protected void doPrepareManifestSummary(
 			_staging.populateLastPublishDateCounts(
 				portletDataContext,
 				new StagedModelType[] {
+					new StagedModelType(
+						DDMDataProviderInstance.class.getName()),
 					new StagedModelType(DDMFormInstance.class.getName()),
 					new StagedModelType(DDMFormInstanceRecord.class.getName())
 				});
 
 			return;
 		}
 
+		ActionableDynamicQuery ddmDataProviderInstanceActionableDynamicQuery =
+			_ddmDataProviderInstanceStagedModelRepository.
+				getExportActionableDynamicQuery(portletDataContext);
+
+		ddmDataProviderInstanceActionableDynamicQuery.performCount();
+
```

Comment:
@brianchandotcom, this code is necessary because the Data Provider is now being considered (counted) by the process of publish to live. Forms and its records are already considered, hence each of these entities have a performCount method too.